src/common/types.ts

```diff
@@ -165,6 +165,7 @@ export type Status =
   | 'notSupportedGame'
   | 'notInstalled'
   | 'installed'
+  | 'ubisoft'
```

Comment:
maybe instead of `ubisoft` we could use something like: 'preparing', 'setup' or something like this because then we can use it for other things as well.

Reply:
I was actually thinking about having more statuses here, so we can be more verbose to the user (like "preparing prefix" for when wineboot is running), maybe we could have another more generic `preparing` that we can use for gog's setup?

I think having more states make it more clear for the user (and also for us to know when something is failing)